lib/main.js

```diff
@@ -45,25 +45,53 @@ Lambda.prototype.run = function (program) {
   var handler = require(process.cwd() + '/' + filename)[handlername];
   var event = require(process.cwd() + '/' + program.eventFile);
 
-  this._runHandler(handler, event);
+  this._runHandler(handler, event, program.runtime);
 };
 
-Lambda.prototype._runHandler = function (handler, event) {
+Lambda.prototype._runHandler = function (handler, event, runtime) {
+
+  var callback = function (err, result) {
+    if (err) {
+      console.log('Error: ' + error);
+      process.exit(-1);
+    }
+    else {
+      console.log('Success:');
+      if (result) {
+        console.log(JSON.stringify(result));
+      }
+      process.exit(0);
+    }
+  };
+
   var context = {
+    isNode43: runtime === "nodejs4.3",
     succeed: function (result) {
-      console.log('succeed: ' + JSON.stringify(result));
-      process.exit(0);
+      if(isNode43) console.log('context.succeed() is deprecated with Node.js 4.3 runtime');
```

Comment:
It is supported for backward compatibility: http://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-using-old-runtime.html#transition-to-new-nodejs-runtime